src/genpy/generator.py

```diff
@@ -79,6 +79,7 @@
 from . generate_struct import reduce_pattern
 from . generate_struct import unpack
 from . generate_struct import unpack2
```

Comment:
This import should be obsolete.

Reply:
I thought about it but if we really want to ensure API compatibility we should not remove the old symbol